libnmstate/nm/applier.py

```diff
@@ -33,7 +33,6 @@
 from . import device
```

Comment:
There is a typo in the commit message; it says `virutal` but should say `virtual`.

Reply:
Done